docs/source/host-collect-analyze/run.md

```diff
@@ -48,6 +63,19 @@ spec:
         collectorName: "docker-logs-etcd"
         command: "sh"
         args: ["-c", "docker logs $(docker ps -a --filter label=io.kubernetes.container.name=etcd -q -l) 2>&1"]
+    # Run command with config gile and collect the file output
+    - run:
+        collectorName: "enriched-audit-logs"
+        # must present on where you run the troubleshoot command
+        command: "enrich-log.sh"
+        args: ["--timeout", "10m", "--output-dir", "$WORKSPACE_OUTPUT"]
```

Comment:
Where would `$WORKSPACE_OUTPUT` come from?

Maybe we do not have to have this example since we have the one right below

Reply:
oops this fell through the crack! actually is `TS_OUTPUT_DIR`

Reply:
yeah this is some outdated stuff let me get rid of this part